python/lsst/meas/extensions/psfex/psfexPsfDeterminer.py

```diff
@@ -376,6 +376,11 @@ def determinePsf(self, exposure, psfCandidateList, metadata=None, flagKey=None):
 
         psf = psfex.PsfexPsf(psfs[0], geom.Point2D(avgX, avgY))
 
+        try:
+            _ = psf.getKernel(psf.getAveragePosition())
+        except pexExcept.InvalidParameterError as e:
+            raise RuntimeError("Failed to determine psfex psf: too few good stars.", e)
```

Comment:
```suggestion
            raise RuntimeError("Failed to determine psfex psf: too few good stars.", e) from None
```
Unless you think the chained exception will be useful.